### PR TITLE
Add scriptManager and zDebugger  (without submodules this time)

### PR DESCRIPTION
**Pull request by Zetaphor**
This PR adds zDebugger and scriptManager to the list of community apps

You can find more information for each projects in its respective repo:

https://github.com/Zetaphor/overte-scriptManager
https://github.com/Zetaphor/overte-zDebugger

### CODE REVIEW

**Comment by Zetaphor** [comment]
This PR includes cash.js and a reduced production build of TailwindCSS, both of which are MIT licensed

File: applications/zDebugger/zDebugger.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+"use strict";
```
Comment (ksuprynowicz) [inline applications/zDebugger/zDebugger.js:1]: Code should have license headers like this one:
https://github.com/overte-org/community-apps/blob/master/applications/aaswitcher/app-aaSwitcher.js

File: applications/zDebugger/zDebugger.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+  function cleanup() {
+    if (appActive) {
+      overlayWebWindow.webEventReceived.disconnect(onWebEventReceived)
+    }
+
+    tablet.removeButton(button)
+  }
+
+  // Script.setInterval(function () {
```
Comment (ksuprynowicz) [inline applications/zDebugger/zDebugger.js:92]: I think we can remove this commented out code

**Review by ksuprynowicz — CHANGES_REQUESTED** [review]
Everything looks fine, the code just needs license headers

**Comment by Zetaphor** [comment]
License headers added and debug removed

File: applications/zDebugger/zDebugger.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
+//
+//  zDebugger.js
+//
+//  Created by Zetaphor, September 21st 2023.
+//  Copyright 2023 Overte e.V.
+//
+//  This application let you setup different Anti-Aliasing setup for HMD and Desktop.
+//  And it switches the configuration when the mode changes.
```
Comment (ksuprynowicz) [inline applications/zDebugger/zDebugger.js:8]: I think you accidentally copied description too :)
Can you add copyright header in HTML file too?

**Review by ksuprynowicz — CHANGES_REQUESTED** [review]

File: applications/zDebugger/zDebugger.js (hunk 3 above)
Comment (Zetaphor) [reply applications/zDebugger/zDebugger.js]: 🤦 That's what I get for speedrunning a commit

**Comment by Zetaphor** [comment]
Headers are updated :)

**Review by ksuprynowicz — APPROVED** [review]
Everything looks fine :)

**Comment by vegaslon** [comment]
this appears to be missing the filed needed to show up in more app

**Comment by Zetaphor** [comment]
You're right, I'll open a new PR

**Pull request merged**